lib/NodeHelpers.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+var Matrix4 = Cesium.Matrix4;
+var Cartesian3 = Cesium.Cartesian3;
+var Quaternion = Cesium.Quaternion;
+
+// A set of helper functions for working with gltf nodes
+// Should be called after convertDagToTree
+
+module.exports = {
+    computeFlatTransformScene: computeFlatTransformScene,
+    getLocalMatrix4: getLocalMatrix4,
+    getAllNodesInScene: getAllNodesInScene,
+    depthFirstTraversal: depthFirstTraversal
+};
+
+// computes flattened Matrix4 transforms for every node in the scene
+// store these matrices for each node in extras._pipeline.flatTransform
+function computeFlatTransformScene(scene, nodes) {
+    var rootNodeIDs = scene.nodes;
+    var scratchMat4 = new Matrix4();
+    var flattenTransformFunction = function(node, parent) {
```

Comment:
> Use function flattenTransformFunction style syntax.

Same comment throughout.

> Also declare this outside of the computeFlatTransformScene function and pass in any needed parameters.

Same comment throughout, but it looks like you'll have to pass the parameter through `depthFirstTraversal` so I'm not 100% sure it is better, but I suspect it is.